source/draw/Image.ooc

```diff
@@ -27,6 +27,11 @@ CoordinateSystem: enum {
 	YUpward = 0x02
 }
 
+TransformMethod: enum {
```

Comment:
Looks like I shouldn't have merged this until I got your answer, but I'll open an issue with those concerns instead.